### PR TITLE
Custom pronouns field

### LINKED ISSUE #4: Custom pronouns within game
As of version 0.1.0, custom pronouns must be set directly inside the config file and there is no way of doing this in-game.

Acceptance criteria:
* There should be a text entry field on the pronouns options screen to allow custom pronouns to be set.

### PR DESCRIPTION
**Pull request by coolreader18**
Resolves #4

### CODE REVIEW

File: src/main/java/mx/ori/pronouns/PronounsScreen.java [hunk 1]
```diff
@@ -101,7 +124,7 @@ protected boolean isFocused() {
 
 
 
-        public class PronounsEntry extends Entry<PronounsEntry> {
+        public class PronounsEntry extends AlwaysSelectedEntryListWidget.Entry<PronounsEntry> {
```
Comment (coolreader18) [inline src/main/java/mx/ori/pronouns/PronounsScreen.java:127]: I'm not sure why my ide wasn't accepting this as it was :/ in any case it's more explicit, at least.

**Comment by coolreader18** [comment]
@ori-sky could I bump this? A server I play on uses my patch and it works well afaict; the guidance from the server owner before was just edit the config file if you want custom pronouns, which isn't super user friendly.

**Comment by ori-sky** [comment]
My apologies, for some reason I never got a notification from GitHub about this PR, or maybe I had missed it somehow. Either way, I've checked this out and it seems to work well. I'll go ahead and merge it.

**Comment by coolreader18** [comment]
Awesome, thank you!

**Pull request merged**